tools/coqdep/coqdep.ml

```diff
@@ -45,7 +47,7 @@ let coqdep () =
     if Sys.file_exists user_contrib
     then Loadpath.add_rec_dir_no_import (Loadpath.add_coqlib_known lst) user_contrib [];
     List.iter (fun s -> Loadpath.add_rec_dir_no_import (Loadpath.add_coqlib_known lst) s [])
-      (Envars.xdg_dirs ~warn:(fun x -> Warning.give "%s" x));
+      (Envars.xdg_dirs ~warn:(fun x -> Feedback.msg_warning (Pp.str x)));
```

Comment:
There are still a few of these, do you intend to convert them to cwarnings too?

Reply:
Possibly, I need to have a closer look.

Reply:
OK, that should do.